PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -3130,7 +3202,57 @@ function Get-SddcDiagnosticInfo
     # Phase 4
     #
 
-    Show-Update "<<< Phase 4 - Compacting files for transport >>>" -ForegroundColor Cyan
+    if ($ZipFiles)
+    {
+        Show-Update "<<< Phase 4 - Compacting files for transport >>>" -ForegroundColor Cyan
+    }
+    else
+    {
+        Show-Update "<<< Phase 4 - Extract cab files + Final Cleanup >>>" -ForegroundColor Cyan
+    }
+
+    if (!$ZipFiles)
+    {
+         Show-update "Extract cab files"

```

Comment:
This belongs in the consumer you're wrapping around Get-SDDC. I took a look and there's a single CAB, one that Get-NetView produces, thats only visible once we stop taking the ZIP from GNV - it makes sense to ask Get-SDDC to not archive the gather as it lands, but with all the specific content ingestion you're writing around it doesn't make sense to have Get-SDDC itself rehydrate pieces/parts of what other tools it calls are gathering. Thats just part of all the other post-processing.

As I recall, the reason I had to have Get-SDDC prefer the GNV ZIP was due to path length issues with the (deep) GNV content if it has taken the unzipped content and ZIP'd it itself within the larger mass of content . Be sure you're not falling foul of that - if there's work needed at GNV to rationalize path depth (... and how much content in what form it should be producing - zipped/unzipped is not strictly a feature) that's a conversation to have with them.

Reply:
Removed. Added this code to wrapper script.